packages/core/src/components/forms/formGroup.tsx

```diff
@@ -50,6 +50,13 @@ export interface IFormGroupProps extends IntentProps, Props {
     /** Label of this form group. */
     label?: React.ReactNode;
 
+    /**
+     * Optional helper text for `label`. The given content will be wrapped in
+     * `Classes.FORM_HELPER_TEXT` and displayed beneath `label`. Helper text
+     * color is determined by the `intent`.
+     */
+    labelHelperText?: React.ReactNode;
```

Comment:
can we call this `sublabel` instead? and make the class name `.bp4-form-group-sublabel` accordingly

Reply:
edit: I would make it two words actually, so the prop is `subLabel` and class `.bp4-form-group-sub-label`

Reply:
That makes sense. Just pushed an update.